content/1_General/Input_Output.mdx

```diff
@@ -1015,27 +1086,30 @@ import java.io.*;
 import java.util.*;
 
 public class Main {
-	CodeSnip{Kattio}
-	
-	static Kattio io;
-	static {
-		try {
-			io = new Kattio("paint");
-		} catch(IOException e) {}
-	}
-	public static void main(String[] args) {
-		int a = io.nextInt(), b = io.nextInt();
-		int c = io.nextInt(), d = io.nextInt();
+	public static void main(String[] args) throws IOException{
+		Kattio io = new Kattio("paint");
+		int a = io.nextInt();
+		int b = io.nextInt();
+		int c = io.nextInt();
+		int d = io.nextInt();
 
 		int[] cover = new int[100];
-		for (int i = a; i < b; i++) cover[i] = 1;
-		for (int i = c; i < d; i++) cover[i] = 1;
-		int ans = 0;
-		for (int i = 0; i < 100; i++) ans += cover[i];
+		for (int i = a; i < b; i++) {
+			cover[i] = 1;
+		}
+		for (int i = c; i < d; i++) {
+			cover[i] = 1;
+		}
 
+		int ans = 0;
+		for (int i = 0; i < cover.length; i++) {
+			ans += cover[i];
+		}
 		io.println(ans);
-		io.close(); // make sure to include this line -- closes io and flushes the output
+		io.close();
 	}
+
+	CodeSnip{Kattio}
```

Comment:
```suggestion
	// CodeSnip{Kattio}
```

Reply:
wait what's the difference
does this break the build or smth